lib/rubygems/commands/bump_command.rb

```diff
@@ -81,7 +83,7 @@ def escape(string)
 
     def commit
       say "Creating commit" unless quiet?
-      system("git commit -m \"Bump to #{@new_version_number}\"")
+      options[:sign] ? system("git commit -S -m \"Bump to #{@new_version_number}\"") : system("git commit -m \"Bump to #{@new_version_number}\"")
```

Comment:
A bit verbose for a one-liner. Could you please break this out with an `if` / `else` ?